cosmic_ray/cli.py

```diff
@@ -56,7 +56,7 @@ def handle_help(config):
     if not command:
         options = OPTIONS
     elif command not in COMMAND_HANDLER_MAP:
-        LOG.error('"{}" is not a valid cosmic-ray command'.format(command))
+        LOG.error('"%s" is not a valid cosmic-ray command' % command)
```

Comment:
Well I also though like you did but the tool says otherwise. I don't have a preference really. If you don't like this particular optimization I will remove it and leave the rest, just let me know what to do.